samples/boards/thingy52_nrf52832/pdm/README.rst

```diff
@@ -0,0 +1,43 @@
+.. _pdm_microphone_sample:
+
+Nordic® Thingy:52 Digital PDM Microphone Sample Application
```

Comment:
Does this application have to be specific to thingy52 ? I am sure that the same should work on nRF52840 Dev Kit with external PDM Mic.